modules/storage-ephemeral-storage-strategy.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * storage/application-ephemeral-storage.adoc
+
+[id='strategy-{context}']
+= Strategy
+
+. Determine expected container ephemeral usage.
++
+If your administrator enabled the ephemeral storage technology preview, 
+determine expected mean and peak container ephemeral storage usage, 
+empirically if necessary, for example, by separate extended. Remember to 
+consider all the processes that might potentially run in parallel in the 
+container. For example, does the main application spawn any ancillary 
+scripts that may require local storage for work files or logging?
+
+. Assess risk for eviction.
++
+Determine the risk appetite for eviction. If the risk appetite is low, 
+set the container to request ephemeral storage according to the expected 
+peak usage plus a percentage safety margin. If the risk appetite is 
+higher, set the container to request ephemeral storage according to the 
+expected mean usage.
+
+. Set container ephemeral storage request.
++
+Set the container ephemeral storage request based on your risk assessment. 
+The more accurately the request represents the application ephemeral 
+storage usage, the better. If the request is too high, cluster and quota 
+usage will be inefficient. If the request is too low, the chances of 
+application eviction increase.
+
+. Set container ephemeral storage limits, if required.
++
+Set container ephemeral storage limits, if required. Setting a limit has 
+the effect of immediately stopping a container process if the combined 
+ephemeral storage usage of all processes in the container exceeds the 
+limit. For example, the container might make unanticipated excess 
+ephemeral storage usage obvious early, that is, fail fast, or the 
+container may stop the processes abruptly.
++
+[IMPORTANT]
+====
+Some {product-title} clusters may require a limit value to be set. Some 
```

Comment:
s/may/might

... How do I know if my cluster requires this? If I'm an admin, how do I set this? I think this note is too vague to be helpful.